### ISSUE TITLE
get-pip failing for embeddable Python on Windows 10

### ISSUE THREAD

**Issue by csboling** [pypa/get-pip#22]
On Windows 10, here is a script snippet that worked until recently to get an embeddable Python install and install pip in it.
```bash
set -ex

rm -rf Python35/
mkdir Python35/
curl https://www.python.org/ftp/python/3.5.2/python-3.5.2-embed-amd64.zip > Python35/python-3.5.2.zip

pushd Python35
unzip python-3.5.2.zip
popd

curl https://bootstrap.pypa.io/get-pip.py > get-pip.py
./Python35/python get-pip.py
```
However, this now gives an error:
```
+ ./Python35/python get-pip.py
ERROR: To modify pip, please run the following command:
C:\...\Python35\python.exe -m pip
```
and this suggestion is not possible because pip is not yet installed.

My current workaround is to replace `curl https://bootstrap.pypa.io/get-pip.py > get-pip.py` with
```
rm -rf get-pip
git clone https://github.com/pypa/get-pip
pushd get-pip
git checkout f135a3e
cp get-pip.py ..
popd
```
since f135a3e seems to be the most recent commit which works.

**Comment by pradyunsg** [comment]
Quoting @pfmoore:

> This is a consequence of pypa/pip#5219 - we have a fix and will be releasing 10.0.1 in the near future. In the meantime, you can work around this by renaming get-pip.py to something without pip in the name (such as gp.py)

**Comment by pradyunsg** [comment]
This should have been fixed.
